content/zh/docs/tasks/access-application-cluster/access-cluster.md

```diff
@@ -371,7 +371,7 @@ You have several options for connecting to nodes, pods and services from outside
       如果服务不能够安全地暴露到互联网，或者服务不能获得节点 IP 端口的访问权限，或者是为了 debug，那么请使用此选项。
     - 代理可能会给一些 web 应用带来问题。
     - 只适用于 HTTP/HTTPS。
-    - 更多详细信息在 [这里]。
+    - 更多详细信息在 [这里](#发现内建服务)。
```

Comment:
Described [here](#manually-constructing-apiserver-proxy-urls)

it seems that is not right.